assembl/static/js/app/utils/tourManager.js

```diff
@@ -27,6 +27,9 @@ var TourManager = Marionette.Object.extend({
       },
 
   initialize: function() {
+    if (Ctx.isAdminApp()) {
+      return;
+    }
```

Comment:
Good idea. Pull request merged, but worth doing separately. (Might do it on develop.)